test/e2e/specs-gutenberg/wp-calypso-gutenberg-focused-launch-spec.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import assert from 'assert';
+import config from 'config';
+import { By } from 'selenium-webdriver';
+import { step } from 'mocha-steps';
+
+/**
+ * Internal dependencies
+ */
+import * as driverManager from '../lib/driver-manager.js';
+import * as dataHelper from '../lib/data-helper.js';
+import * as driverHelper from '../lib/driver-helper';
+import LoginFlow from '../lib/flows/login-flow.js';
+import CreateSiteFlow from '../lib/flows/create-site-flow.js';
+import DeleteSiteFlow from '../lib/flows/delete-site-flow.js';
+import MyHomePage from '../lib/pages/my-home-page';
+import NavBarComponent from '../lib/components/nav-bar-component.js';
+import GutenbergEditorComponent from '../lib/gutenberg/gutenberg-editor-component';
+
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
+
+const host = dataHelper.getJetpackHost();
+const screenSize = driverManager.currentScreenSize();
+
+describe( `[${ host }] Calypso Gutenberg Editor: Focused launch on (${ screenSize })`, function () {
+	this.timeout( mochaTimeOut );
+	let driver;
+
+	before( 'Start browser', async function () {
+		this.timeout( startBrowserTimeoutMS );
+		driver = await driverManager.startBrowser();
+	} );
+
+	describe( 'Launch a free site', function () {
+		const siteName = dataHelper.getNewBlogName();
+
+		before( 'Can log in', async function () {
+			const loginFlow = new LoginFlow( driver );
+			await loginFlow.login();
+		} );
+
+		step( 'Can create a free site', async function () {
+			await new CreateSiteFlow( driver, siteName ).createFreeSite();
+		} );
```

Comment:
Since Focused Launch is rolled-out only for `/new` sites at the moment, shouldn't we use `gutenboarding-flow` to create the site? See https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/lib/flows/gutenboarding-flow.js#L50

Reply:
Yup, moved over. It was `/start` because FL was created in start in the beginning haha. :)